oak_functions/examples/abitest/backend/src/main.rs

```diff
@@ -0,0 +1,72 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! Oak Functions ABI test backend server.
```

Comment:
What is this backend used for? Is it not possible to use the existing Mock server for this? Even if we need it as a separate binary, it seems it should be based on the same functionality in Rust. In fact at that point it may as well be a binary that is configurable with a flag pointing to the file(s) to serve.

In particular, it should not serialize the entries on the fly, it should just serve a static file that was created from the entries beforehand (it's fine to check in such a file, with instructions on how to re-generate it, perhaps extending the lookup data generator binary first providing a new mode that generates a file with a single entry, provided via flags). This way it would also test that we keep compatibility with the lookup data format, since we would need to re-generate it if we change the format (while in the current test we would not notice).